src/backend/commands/matview.c

```diff
@@ -1749,3 +1666,79 @@ apply_delta(Oid matviewOid, Oid tempOid_new, Oid tempOid_old,
 	if (SPI_finish() != SPI_OK_FINISH)
 		elog(ERROR, "SPI_finish failed");
 }
+
+/*
+ * get_materializedview_query - get the Query from a matview's _RETURN rule.
+ */
+Query *
+get_materializedview_query(Relation matviewRel, bool concurrent)
+{
```

Comment:
This might be a minute thing, but how about get_materialized_view_query or get_matview_query?

Reply:
Ok, I will change function name as get_matview_query.